content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -0,0 +1,296 @@
+
+In this step, you will use CMake to build either a monolithic or a non-monolithic project and specify the `INSTALL` target and `release` configuration. This handles several tasks:
+
+- Creates an `install` directory in your project directory that will contain the project's release layout.
+
+- Bundles your game content into an `engine.pak` file by packaging all of the product assets in your `Cache/pc` directory.
+
+- Builds the Game Launcher and other supporting files.
+
+{{< todo >}}
+The Game Launcher may fail to load your assets if the `engine.pak` file is over 4 GB. This issue will be resolved when development on the Asset Bundler is stable.
+{{< /todo >}}
+
+### Using source engine
+ 
+With the source engine, you must use a project-centric configuration to create a project game release layout. In a project-centric configuration, the CMake source directory is in the project build directory.
+
+The source engine supports non-monolithic projects by default. As listed under [Prerequisites](#prerequisites), you should have already created a Visual Studio project for the engine and registered the engine.
+
+#### Non-monolithic
+
+1. Use CMake to invoke Visual Studio to build your non-monolithic project. Specify the `INSTALL` target and `release` configuration.
+
+    ```cmd
+    cd C:/o3de-projects/MyProject
+    cmake --build build/windows_vs2019 --target INSTALL --config release. 
+    ```
+
+The result is a project release install directory that's located in your project's directory at `install/bin/Windows/release/Default`. Your packaged assets are located in `Cache/pc` in your project release install directory.
+
+#### Monolithic
+
+1. Use CMake to create a Visual Studio project for a monolithic project. Specify a new CMake build directory, `build/windows_mono`, in your project directory to seperate it from your non-monolithic build directory. Specify a monolithic build by enabling the `-D` option, `LY_MONOLITHIC_GAME`.
+
+    ```cmd
+    cd C:/o3de-projects/MyProject
+    cmake -B build/windows_mono -S. -G "Visual Studio 16" -DLY_3RDPARTY_PATH=C:\o3de-packages -DLY_MONOLITHIC_GAME=1
+    ```
+
+2. Use CMake to invoke Visual Studio to build your monolithic project. Specify the `INSTALL` target and `release` configuration. This generates a Game Launcher and packages your project's assets in the `<project>/Cache` directory into `engine.pak`.
+
+    ```cmd
+    cd C:/o3de-projects/MyProject
+    cmake --build build/windows_mono --target INSTALL --config release
+    ```
+
+The result is a project release install directory that's located in your project's directory at `install/bin/Windows/release/Monolithic`. Your packaged assets are located in `Cache/pc` in your project release install directory. 
```

Comment:
Similier to the Non-Monolithic Project Game Release Layout, we should clarify that this install directory is the value of the `CMAKE_INSTALL_PREFIX` cache variable which default to the `<project-root>/install/bin/Windows/release/Monolithic` if not specified.

Reply:
+1,  it's good to point this out because likely users will want to set `CMAKE_INSTALL_PREFIX` to put the output in a custom location on their disk